doc/conf.py

```diff
@@ -7,4 +7,5 @@
 doxylink = {}
 exclude_patterns = ["changes/*"]
 
-intersphinx_mapping['networkx'] = ('https://networkx.org/documentation/stable/', None)  # noqa: F405
+intersphinx_mapping["networkx"] = ("https://networkx.org/documentation/stable/", None)  # noqa: F405
+intersphinx_mapping["lsst"] = ("https://pipelines.lsst.io/v/daily/", None)  # noqa
```

Comment:
No `: F405` for this one?

Reply:
Probably. I copied it from a different conf file.